app/views/patients/_fulfillment.html.erb

```diff
@@ -42,6 +42,11 @@
           <% end %>
         </div>
       </div>
+      <div class="row">
+        <div class="col-sm-12">
+          <p>Patient will be archived on <%= patient.archive_date %>
```

Comment:
I would suggest slapping a `display_date` on this, e.g. `patient.archive_date.display_date`. (if it doesn't work immediately then it's because I spaced on something so nevermind, haha)

Reply:
didn't work - but I don't actually know why not, as it works just a bit above here

Reply:
my only sadness now with this is that when you click `audited` right nearby, the date won't update until a page reload.

Reply:
would it be worthwhile or overkill (or bad UX?) to add some JS code here to trigger a submit on the checkbox click, so the user doesn't need to reload the page for the change to take effect?

Reply:
Now it does!